.gitignore

```diff
@@ -8,6 +8,9 @@
 !Dockerfile
 !.dockerignore
 !pyproject.toml
+!docker-compose.yml
+!conf.d/flaskapp.conf
+!env_file
```

Comment:
`env_file` was specified here but wasn't added to the changelist. Was this intended?